business_requirement_deliverable/models/business.py

```diff
@@ -141,10 +141,38 @@ def product_id_change(self):
             description = product.name
             uom_id = product.uom_id.id
             unit_price = product.list_price
+        if self.business_requirement_id.project_id.pricelist_id and \
```

Comment:
There is not pricelist in project AFAIK.
Anyway it needs to be a separate pricelist for estimation